python/paddle/fluid/tests/unittests/mkldnn/test_reshape_bf16_op.py

```diff
@@ -28,7 +28,7 @@
 class TestReshapeBf16Op(OpTest):
     def setUp(self):
         self.op_type = "reshape2"
-        self.use_mkldnn = True
+        self.use_mkldnn = False
```

Comment:
Good point. It should be False because reshape2 doesn't even have `use_mkldnn` attribute.